app/Wiki.php

```diff
@@ -57,6 +57,8 @@ class Wiki extends Model
         'deleted_at',
     ];
 
+    protected $perPage = 20;
```

Comment:
Not sure if this override is even needed. It defaults to 15 and I mainly put this value here for testing purposes. In the real world, I would expect the UI to always pass a `per_page` query param to tell the endpoint about its preferences.